awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsAttributeKeys.java

```diff
@@ -56,4 +56,13 @@ private AwsAttributeKeys() {}
   static final AttributeKey<String> AWS_QUEUE_NAME = AttributeKey.stringKey("aws.queue.name");
   static final AttributeKey<String> AWS_STREAM_NAME = AttributeKey.stringKey("aws.stream.name");
   static final AttributeKey<String> AWS_TABLE_NAME = AttributeKey.stringKey("aws.table.name");
+  static final AttributeKey<String> AWS_AGENT_ID = AttributeKey.stringKey("aws.bedrock.agent_id");
+  static final AttributeKey<String> AWS_KNOWLEDGEBASE_ID =
+          AttributeKey.stringKey("aws.bedrock.knowledgebase_id");
+  static final AttributeKey<String> AWS_DATASOURCE_ID =
+          AttributeKey.stringKey("aws.bedrock.datasource_id");
+  static final AttributeKey<String> AWS_GUARDRAIL_ID =
+          AttributeKey.stringKey("aws.bedrock.guardrail_id");
+  static final AttributeKey<String> AWS_BEDROCK_RUNTIME_MODEL_ID =
```

Comment:
should it be AWS_BEDROCK_MODEL_ID? All foundation models belongs to Bedrock instead of Bedrock runtime.

Reply:
Change to GEN_AI_REQUEST_MODEL